Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -353,6 +353,36 @@ public void StartClient()
             OnStartClient();
         }
 
+        /// <summary>
+        /// This starts a network client. It uses the networkAddress and networkPort properties as the address to connect to.
+        /// <para>This makes the newly created client connect to the server immediately.</para>
+        /// </summary>
+        /// <param name="uri">location of the server to connect to</param>
+        public void StartClient(Uri uri)
```

Comment:
maybe this function should set networkmanager.address to uri.address in case someone access it later?

Reply:
do you want to use uri.ToString() as the address or you want to user uri.Host ?

Reply:
I meant host. Like you do below for transport default behaviour

Reply:
done